project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject puppetlabs/lein-ezbake "1.4.1-SNAPSHOT"
+(defproject puppetlabs/lein-ezbake "1.5.0-SNAPSHOT"
```

Comment:
I looked at the previous commits in the repo that prepared for releases and they do this.

The jenkins job will just strip off the `-SNAPSHOT`, do the release (so we get `1.5.0`), then update the string to `1.5.1-SNAPSHOT`.